x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -512,6 +531,12 @@ export const EndpointList = () => {
           onChange={onTableChange}
           loading={loading}
           rowProps={setTableRowProps}
+          sorting={{
```

Comment:
in order to avoid needless re-renders, I would suggest memoizing the object provided to this `sorting` prop.

Reply:
done. also memoized some other variables passed as props, because there were too much re-renders on e.g. opening the details flyout.

there are still a lot of re-renders, but now it doesn't go down to the table that much

eb3413f2e3effd1cd8220624cc43fb0f33b155f4